x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -430,3 +430,12 @@ export const RULES_MANAGEMENT_FEATURE_TOUR_STORAGE_KEY =
 
 export const RULE_DETAILS_EXECUTION_LOG_TABLE_SHOW_METRIC_COLUMNS_STORAGE_KEY =
   'securitySolution.ruleDetails.ruleExecutionLog.showMetrics.v8.2';
+
+/**
+ * Error codes that can be thrown during _bulk_action API dry_run call and be processed and displayed to end user
+ */
+export enum BULK_ACTIONS_DRY_RUN_ERR_CODE {
+  IMMUTABLE = 'IMMUTABLE',
+  MACHINE_LEARNING_AUTH = 'MACHINE_LEARNING_AUTH',
+  MACHINE_LEARNING_INDEX_PATTERN = 'MACHINE_LEARNING_INDEX_PATTERN',
+}
```

Comment:
It's written in this way, because values represent const error codes, that will passed through API and later interpreted in different cases: as showing error, building query filter

Example from the codebase that handles similar case:
https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/detections/components/rules/eql_query_bar/validators.ts#L18-L21

Reply:
renamed to `BulkActionsDryRunErrCode`